### ISSUE TITLE
Evince command injection vulnerability in CBT handler

### ISSUE THREAD

**Issue by jollyrogers88** [linuxmint/xreader#150]
Is [CVE-2017-1000083](https://bugzilla.gnome.org/show_bug.cgi?id=784630) fixed here?

**Comment by hadess** [comment]
It's not. I didn't know this project existed, which is why I didn't report it here (I reported it for MATE's atril). See https://github.com/mate-desktop/atril/issues/257

**Comment by muelli** [comment]
I'm willing to bet that this isn't fixed with e2111a992bf5d0d4f31ad2070f84d432b29872b2 because GNU tar seems to not require the `=`.
```
tar -xf archive.tar  \
         --checkpoint=1 --checkpoint-action exec='printf "%04d in %32s\n" $TAR_CHECKPOINT $TAR_ARCHIVE'
```
works well for me.

In general, I'd be hesitant to filter on that option. The next tar option allowing to run programs is waiting for you. Maybe it's `-I` already, but maybe we have to wait for tar to introduce a new one.

How many other Evince forks exist that have tried to fix this issue but failed?
